content/agent/basic_agent_usage/windows.md

```diff
@@ -19,15 +19,36 @@ further_reading:
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
+This page outlines the basic functionality of the Windows Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
 
 ### Starting and Stopping the Agent
 
-The execution of the Agent is controlled by a Windows service.
+The execution of the Agent is controlled by a Windows service. 
+There are many items that can be configured on the command line. Each configuration item is added as an install property to the command line, for instance the following commands install the Agent, and configure the Agent configuration file with the `<DATADOG_API_KEY>`, as well as setting the `<HOSTNAME>` and tags.
```

Comment:
There are many items that can be configured on the command line. Each configuration item is added as an install property to the command line. For instance, the following commands install the Agent, configure the Agent configuration file with the `<DATADOG_API_KEY>`, and set the `<HOSTNAME>` and tags.